pkg/driver/controller.go

```diff
@@ -383,6 +401,25 @@ func isValidVolumeCapabilities(volCaps []*csi.VolumeCapability) bool {
 	return foundAll
 }
 
+// Check if the volume is shareable
+func isShareableVolume(volCaps []*csi.VolumeCapability) bool {
```

Comment:
good to add some unit test cases for this method

Reply:
Unit tests added.